Clarify input size constraints for gpt-image-2 images (#779)
MicrosoftDocs/azure-ai-docs: articles/foundry/openai/how-to/dall-e.md

- Aspect ratio up to 3:1.
- Total pixel count between 655,360 and 8,294,400.

> [!NOTE]

> > The constraints apply only to explicitly specified sizes. If size=auto, the generated image may have dimensions that do not satisfy these constraints (for example, an edge may not be a multiple of 16 pixels).

Reviewer: ```suggestion
> The sizing constraints only apply when you specify a size. If `size=auto`, the generated image may have dimensions that do not satisfy these constraints (for example, an edge might not be a multiple of 16 pixels).
```
